llvm/test/CodeGen/NVPTX/convert-sm100a.ll

```diff
@@ -0,0 +1,290 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=nvptx64 -mcpu=sm_100a -mattr=+ptx86| FileCheck --check-prefixes=CHECK %s
```

Comment:
Nit: I would at a space before the `|`. Also `CHECK` is the default prefix so `--check-prefixes=CHECK` should be removed.